modules/fee-market/src/lib.rs

```diff
@@ -503,7 +503,8 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 	}
 
 	fn collateral_to_order_capacity(collateral: BalanceOf<T, I>) -> u32 {
-		(collateral / T::CollateralPerOrder::get()).saturated_into::<u32>()
+		// If the `CollateralPerOrder` is zero, the maximum order capacity is `collateral`.
```

Comment:
If the `CollateralPerOrder` is zero, we should force return 0. which means all relayers have no available order capacity. the fee market is not available in this situation until we fix it in the runtime.

Reply:
If zero means not allow to relay, better to document that.